source/MaterialXRuntime/RtStage.cpp

```diff
@@ -35,6 +35,22 @@ void RtStage::removeReference(const RtToken& name)
     data()->asA<PrvStage>()->removeReference(name);
 }
 
+RtStage RtStage::getReference(const RtToken& name) const
```

Comment:
To follow the convention used in other runtime API we could have the following reference access methods:

```
size_t numReferences() const;
RtObject getReference(size_t index) const;
RtObject findReference(const RtToken& name) const;
```
Then we can iterate on all the references as well as finding them by name.

Reply:
Will add.